kujira/assetlist.json

```diff
@@ -233,6 +233,65 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdt.svg"
       }
+    },
+    {
+      "description": "whLOCAL on Kujira",
+      "denom_units": [
+        {
+          "denom": "ibc/DE92DB84EA7E8CB5EC76BC395CF634D6483B67414AD28BEF3F71EBFD6A780D36",
+          "exponent": 0
+        },
+        {
+          "denom": "whLOCAL",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/DE92DB84EA7E8CB5EC76BC395CF634D6483B67414AD28BEF3F71EBFD6A780D36",
+      "name": "whLOCAL",
+      "display": "whLOCAL",
+      "symbol": "whLOCAL",
+      "traces": [
+        {
+          "type": "ibc",
```

Comment:
The type here should be 'ibc-cw20', which also requires the port properties

Reply:
Just applied those changes hope that's ok